requirements.txt

```diff
@@ -1,7 +1,9 @@
+python-dateutil>=2.1,<=2.8.0
+numpy>=1.17.3
+kfserving>=0.2.1.1
 docker>=3.4.1
```

Comment:
It's 0.2.1.1? Thought it was supposed to be 0.2.1.0?

Reply:
We released 0.2.1.0, but failed to integration with fairing, caused by Azure changes, I send a PR to fixed that, and then released 0.2.1.1, you reviewed that PR. If not release 0.2.1.1, we cannot intergrate with fairing. See here for details https://github.com/kubeflow/kfserving/pull/534